openzwave-sys/build.rs

```diff
@@ -1,13 +1,23 @@
 extern crate gcc;
+extern crate pkg_config;
 
 fn main() {
+    let libopenzwave = pkg_config::Config::new()
+        .statik(true)
```

Comment:
do we want a static build ? not sure it's necessary if we have pkg_config.
(sorry if I misunderstood this line)

Note: in the future we'll want to fetch openzwave as a submodule so that we can crosscompile it (if we are in a crosscompiling setup). At that time I think we'll also need to change how we include it as `#include <openzwave/xxx.h>` does not work with the main repo :(